demo-app/client/src/tests/delete/delete-order.service.spec.ts

```diff
@@ -25,13 +24,13 @@ describe(`DeleteOrderService`, () => {
       inject([DeleteOrderService, HttpTestingController],
         (service: DeleteOrderService, backend: HttpTestingController) => {
 
-        service.order({orderId: '123e4567-e89b-12d3-a456-426655440000'}).subscribe();
+          service.order({ orderId: '123e4567-e89b-12d3-a456-426655440000' }).subscribe();
 
-        backend.expectOne((req: HttpRequest<any>) => {
-          return req.method === 'DELETE'
-            && req.url === '/api/order/123e4567-e89b-12d3-a456-426655440000';
-        });
-      }),
+          backend.expectOne((req: HttpRequest<any>) => {
+            return req.method === 'DELETE'
+              && req.url.endsWith('/api/order/123e4567-e89b-12d3-a456-426655440000');
```

Comment:
Please, check for equality, and add a test for the new feature (overridden base url).

Reply:
I did it in that way because the === check with override base url needs more code in the test, cause when you are testing custom injections you must to do it manually.